tests/dataplicity/m2m/wsclient/test_channelfile.py

```diff
@@ -0,0 +1,33 @@
+from dataplicity.m2m.wsclient import ChannelFile
+
+
+class FakeClient(object):
+    """ The ChannelFile class calls
+        `self.client.channel_write` inside, therefore this simple class can
+        act as a client representative.
+    """
+    def __init__(self):
+        self.buffer = ''
```

Comment:
ChannelFile uses sys.stdout underneath and when I do that, I get:
```
>       sys.stdout.write(data)
E       TypeError: string argument expected, got 'bytes'
```

I suppose that the underlying implementation could, theoretically use sys.stdout.write(data.decode('utf-8')) but this is tricky as those are binary data and we don't know for sure how they were encoded. The other solution of course would be to do:

```sys.stdout.write(repr(data))```

but this would mean that the code for ChannelFile would change. If you're fine with the rewrite, I can push the change